README.md

```diff
@@ -42,6 +42,9 @@ npm run build
 ```
 
 ### Test
+
+Install docker-compose
+
```

Comment:
Can we remove this text now that `setupIpfs` script is changed?

Reply:
May be to change it to `Install docker`?